src/xdocs/checks/annotation/missingoverride.xml

```diff
@@ -67,86 +67,91 @@
         <source>
-class Test extends SuperClass {
+class Example1 extends ParentClass1 {
 
-    /** {@inheritDoc} */
-    @Override
-    public void test1() { // OK
+  /** {@inheritDoc} */
+  @Override
+  public void test1() { // OK
 
-    }
+  }
 
-    /** {@inheritDoc} */
-    public void test2() { // violation, should be annotated with @Override
+  /** {@inheritDoc} */
+  public void test2() { // violation, 'include @java.lang.Override'
 
-    }
+  }
 
-    /** {@inheritDoc} */
-    private void test3() { // violation, using the @inheritDoc tag on private method
+  /** {@inheritDoc} */
+  // violation below '{@inheritDoc} tag is not valid at this location.'
+  private void test3() {
 
-    }
+  }
 
-    /** {@inheritDoc} */
-    public static void test4() { // violation, using the @inheritDoc tag on static method
+  /** {@inheritDoc} */
+  // violation below '{@inheritDoc} tag is not valid at this location.'
+  public static void test4() {
 
-    }
+  }
 }
         </source>
-        <p>
+        <p id="Example2-config">
           To configure the check for the <code>javaFiveCompatibility</code>
           mode:
         </p>
         <source>
-&lt;module name=&quot;MissingOverride&quot;&gt;
-  &lt;property name=&quot;javaFiveCompatibility&quot;
-      value=&quot;true&quot;/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MissingOverride&quot;&gt;
+      &lt;property name=&quot;javaFiveCompatibility&quot;
+                value=&quot;true&quot;/&gt;
+    &lt;/module&gt;
+  &lt;/module&gt;
 &lt;/module&gt;
         </source>
-        <p>Example:</p>
+        <p id="Example2-code">Example:</p>
         <source>
-class Test1 {
-
-    /** {@inheritDoc} */
-    public void equals() { // violation, should be annotated with @Override
-
-    }
+class Example2 {
+  /** {@inheritDoc} */
+  public boolean equals(Object o) { // violation, 'include @java.lang.Override'
+    return o == this;
+  }
 }
 
-interface Test2 {
+interface B {
 
-    /** {@inheritDoc} */
-    void test(); // violation, should be annotated with @Override
+  /** {@inheritDoc} */
+  void test(); // violation, 'include @java.lang.Override'
 }
 
-class Test3 extends SuperClass {
+class C extends ParentClass2 {
+  /** {@inheritDoc} */
+  public void test() { // OK, is ignored because class extends other class
 
-    /** {@inheritDoc} */
-    public void test() { // OK, is ignored because class extends other class
+  }
+}
 
-    }
+class D implements B {
+  /** {@inheritDoc} */
+  public void test() { // OK, is ignored because class implements interface
+  }
 }
 
-class Test4 implements SuperInterface {
+class E {
+  Runnable r = new Runnable() {
 
     /** {@inheritDoc} */
-    public void test() { // OK, is ignored because class implements interface
-
+    public void run() { // OK, is ignored because class is anonymous class
     }
-}
-
-class Test5 {
-    Runnable r = new Runnable() {
-
```

Comment:
Please restore this example. Or explain why you think it better to drop it.

Reply:
I haven't removed it I just renamed it under the class name _**E**_ as it was not a Test, so naming it Test would not be the best choice.

Reply:
Ok, let me read html version of site to see it more clear

Old: https://checkstyle.org/checks/annotation/missingoverride.html#Examples

Reply:
New: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/529e8c3_2023195343/checks/annotation/missingoverride.html#Examples

Looks good